packages/frontend-app/src/app/components/Card/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+
+import "./index.scss";
+
+interface IProps {
+  src: string;
+  title: string;
+  artist: string;
+}
+
+export const Card: React.FC<IProps> = ({
+  src,
+  title = "I told them...",
+  artist = "Burna Boy",
+}) => {
+  const [isHovered, setIsHovered] = useState(false);
+
+  const handleMouseOver = () => {
+    setIsHovered(true);
+  };
+
+  const handleMouseOut = () => {
+    setIsHovered(false);
+  };
+
+  return (
+    <div
```

Comment:
The card is clickable so please wrap this entire component with a `Link` tag. Then we need an extra link prop